code/__defines/mobs.dm

```diff
@@ -127,6 +127,7 @@
 #define BP_AUG_FUEL_CELL    "integrated fuel cell"
 #define BP_AUG_AIR_ANALYZER "integrated air analyzer"
 #define BP_AUG_LANGUAGE     "integrated language processor"
+#define BP_AUG_NLOM         "nlom augment"
```

Comment:
might wanna make this sound better, because iirc this is the name that pops up during surgery?